### DISCUSSION TITLE
Fridge / Freezer Thermostat control

### DISCUSSION THREAD

**Discussion by jgstroud** [thoukydides/homebridge-homeconnect#182] [General]
I've added support for the fridge and freezer thermostats, but I see this project does not accept pull requests.  Is there any interest in this patch?

**Comment by thoukydides** [comment]
Sorry, no. I am not willing to add Thermometer services to this plugin (regardless of whether it is for Cooling or Cooking appliances).

I use Siri to control by heating system and to check the temperatures in my house. Adding Thermostat services that don’t measure/control the room temperature would result in Siri treating them as part of the house heating controls. Asking Siri for the house/room temperature would include those in the reported min/max. Similarly asking Siri to adjust the room target temperature would also set those controls.

This was previously discussed in #58 and my attitude has not changed.

Perhaps Apple will add new HomeKit Services with different semantics to support Matter 1.2…

**Comment by jgstroud** [comment]
Valid point.  Seems like we could just add a checkbox in the config to determine whether or not to include the additional controls.  But like you have stated before, it's a personal project for you and it doesn't suite your needs.

Thanks for open sourcing your work.
